assets/src/edit-story/elements/video/save.js

```diff
@@ -39,9 +39,22 @@ function VideoSave( { autoPlay, id, mimeType, src, width, height, x, y, rotation
 		id: 'el-' + id,
 	};
 
-	const style = getCommonAttributes( { width, height, x, y, rotationAngle } );
+	const style = isBackground ?
+		{
+			width: '100%',
+			height: '100%',
+			top: 0,
+			left: 0,
+			rotationAngle: 0,
+		} :
+		getCommonAttributes( { width, height, x, y, rotationAngle } );
+
+	if ( isBackground ) {
```

Comment:
Shouldn't we respect the existing `autoPlay` param passed to this component? Maybe change that instead.

Reply:
Good question, not sure. I suppose that's a UX question. It seems like if a video is set as background then it should always autoplay, however, when the background is unset again for that element then I'd assume that the previous `autoplay` setting would remain as it was before.

Let's confirm with UX, this could also be addressed in #45 once the UX for the Video element is confirmed.

Reply:
For some reason I was always under the impression that all videos always autoplay in stories. I'm looking at the [doc here](https://amp.dev/documentation/examples/multimedia-animations/supported_media_types_in_amp_stories/?format=stories), which states:

> Note that amp-video extension has different validation rules when used within an AMP Story—both the autoplay and poster attributes are required.

And consequently, I don't think `controls` are allowed or make much sense.

Reply:
Hmm, I think we've had that conversation before with the previous Stories editor as well already.

Yep, here's a relevant PR:
https://github.com/ampproject/amp-wp/pull/2255

So yes, it looks like the controls don't really make sense anyway. Will leave this for resolving/looking into in #45 and related issues and merge as-is for the background media.